docs/source/examples/cirq-ibmq-backends.myst

```diff
@@ -0,0 +1,253 @@
+    """Returns the expectation value to be mitigated.
+
+    Args:
+        circuit: Circuit to run.
+        shots: Number of times to execute the circuit to compute the expectation value.
+    """
+    # convert from Cirq circuit to Qiskit circuit.
+    circuit = to_qiskit(cirq_circuit) 
+    
+    if USE_REAL_HARDWARE:
+        # Run the circuit on hardware
+        job = qiskit.execute(
+            experiments=circuit,
+            backend=backend,
+            optimization_level=0,  # Important to preserve folded gates.
+            shots=shots
+        )
+    else:
+        # Simulate the circuit with noise
+        noise_model = initialized_depolarizing_noise(noise_level=0.02)
+        job = qiskit.execute(
+            experiments=circuit,
+            backend=qiskit.Aer.get_backend("qasm_simulator"),
+            noise_model=noise_model,
+            basis_gates=noise_model.basis_gates,
+            optimization_level=0,  # Important to preserve folded gates.
+            shots=shots,
+        )
+
+    # Convert from raw measurement counts to the expectation value
+    counts = job.result().get_counts()
+    if counts.get("0") is None:
+        expectation_value = 0.
+    else:
+        expectation_value = counts.get("0") / shots
+    return expectation_value
+```
+
+
+At this point, the circuit can be executed to return a mitigated expectation value by running Mitiq's {func}`.zne.execute_with_zne` function as follows.
+
+```{code-cell} ipython3
+unmitigated = cirq_ibm_executor(cirq_circuit)
+mitigated = zne.execute_with_zne(cirq_circuit, cirq_ibm_executor)
+print(f"Unmitigated result {unmitigated:.3f}")
+print(f"Mitigated result {mitigated:.3f}")
+```
+
+As long as a circuit and a function for executing the circuit are defined, the ``zne.execute_with_zne`` function can
+be called as above to return zero-noise extrapolated expectation value(s).
+
+
+## Options
+
+
+Different options for noise scaling and extrapolation can be passed into the ``zne.execute_with_zne`` function.
+By default, noise is scaled by locally folding gates at random, and the default extrapolation is Richardson.
+
+To specify a different extrapolation technique, we can pass a different ``Factory`` object to ``execute_with_zne``. The
+following code block shows an example of using linear extrapolation with five different (noise) scale factors.
+
+```{code-cell} ipython3
+linear_factory = zne.inference.LinearFactory(scale_factors=[1.0, 1.5, 2.0, 2.5, 3.0])
+mitigated = zne.execute_with_zne(cirq_circuit, cirq_ibm_executor, factory=linear_factory)
+print(f"Mitigated result {mitigated:.3f}")
+```
+
+To specify a different noise scaling method, we can pass a different function for the argument ``scale_noise``. This
+function should input a circuit and scale factor and return a circuit. The following code block shows an example of
+scaling noise by folding gates starting from the left (instead of at random, the default behavior for
+``zne.execute_with_zne``).
```

Comment:
```suggestion
To specify a different noise scaling method, we can pass a different function for the argument ``scale_noise``. This
function should input a circuit and scale factor and return a circuit. The following code block shows an example of
scaling noise by global folding (instead of local folding, the default behavior for
``zne.execute_with_zne``).
```